### PR TITLE
whitelist fs

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist fs

### Linked issues
Progresses #1901 

### Functionality
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -4048,6 +4048,65 @@
...
+    "fs.opener.tarfs": [],
+    "fs.opener.tempfs": [],
+    "fs.opener.zipfs": [],
+    "fs.osfs": [],
+    "fs.path": [],
+    "fs.permissions": [],
+    "fs.subfs": [],
+    "fs.tarfs": [],
+    "fs.tempfs": [],
+    "fs.test": [],
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:4100]: Remove

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -4048,6 +4048,65 @@
   "frozenlist": {
     "frozenlist": []
   },
+  "fs": {
+    "fs": [],
+    "fs._bulk": [],
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:4053]: Same question about hidden modules

**Review by JCZuurmond — COMMENTED** [review]
See comments

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:4100]: Done

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:4053]: Already answered I believe ?

Comment (JCZuurmond) [reply src/databricks/labs/ucx/source_code/known.json:4053]: yes

**Review by JCZuurmond — APPROVED** [review]

**Pull request merged**